mmdet/models/dense_heads/anchor_head.py

```diff
@@ -565,37 +590,14 @@ def get_bboxes(self,
             assert len(
                 img_metas
             ) == 1, 'Only support one input image while in exporting to ONNX'
-            img_shapes = img_metas[0]['img_shape_for_onnx']
```

Comment:
This shape is for supporting dynamic shape of ONNX. Should consider this after removing.